### PR TITLE
Ei/slot config migration

### PR DESCRIPTION
**Pull request by emma-imber**
## What does this change?
Change the slot-config file to use sizes defined in commerical-core - reducing duplication of size definitions
## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?
Reducing the duplication of size definitions in the commercial code - trying to keep sizes defined in one place
### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts [hunk 1]
```diff
@@ -47,55 +48,58 @@ const getSlots = (contentType: string): HeaderBiddingSlot[] => {
 		{
 			key: 'right',
 			sizes: hasShowcase
-				? [[300, 250]]
+				? [[adSizes.mpu.width, adSizes.mpu.height]]
```
Comment (arelra) [inline static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts:51]: I'm just wondering if we can treat `adSizes.whatever` as an array so:

```suggestion
				? [adSizes.mpu]
```

@Jakeii did some magic around this i.e. `AdSize extends Array` this so might be worth trying out.

**Review by arelra — APPROVED** [review]
A great improvement on the old hardcoded values - and so much easier to tell which sizes are being used!

Thanks for following up on this.

File: static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts (hunk 1 above)
Comment (emma-imber) [reply static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts:51]: Thanks Ravi! I got a type error when I tried to do this originally so I'll have a chat with Jake and see what we come up with

**Review by Jakeii — APPROVED** [review]
Nicely Done!

File: static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts (hunk 1 above)
Comment (arelra) [reply static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.ts]: Nice update! My approval still stands so feel free to merge :)

**Pull request merged**